scripts/before_compile.js

```diff
@@ -45,8 +45,12 @@ module.exports = function(ctx) {
   const ignore = ['node_modules'];
   const sentryCli = new SentryCli(configFile);
 
-  const projectRoot = ctx.opts.projectRoot || '';
-  const buildPaths = ctx.opts.platforms.map(p => path.join(projectRoot, 'platforms', p, 'www'));
+  const buildPaths = ctx.opts.paths || ctx.opts.platforms.map(p => {
+    const apiPath = path.join('platforms', p, 'cordova', 'Api.js');
+    const PlatformApi = require(apiPath);
```

Comment:
Can we wrap this require in a try/catch, just in case this file isn’t here so this thing doesn’t blow up.

Otherwise LGTM

Reply:
> Can we wrap this require in a try/catch, just in case this file isn’t here so this thing doesn’t blow up.
> 
> Otherwise LGTM

Ok, updated PR to include a check to make sure the file exists before trying to `require()` it, and also updated to ensure the absolute path was being used to prevent "Cannot find module" errors.